modules/chainBuy.lua

```diff
@@ -121,8 +121,22 @@ function ChainBuy:ShowWindow()
 	end
 
 	local window = StdUi:Window(UIParent, 'Chain Buy', 400, 300);
-	StdUi:GlueBelow(window, AuctionFrame, 0, -40, 'CENTER');
+	if AuctionFaster.db.chain_buy.moved then
```

Comment:
Wanted to extract below conditional to a function since it is basically being used twice... here and on line 128